### ISSUE TITLE
Failed to get convolution algorithm

### ISSUE THREAD

**Issue by SumNeuron** [tensorflow/tensorflow#32519]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): yes / no
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): 16.04 LTS Ubuntu
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: n/a
- TensorFlow installed from (source or binary): conda
- TensorFlow version (use command below): 1.14
- Python version: 2.6
- Bazel version (if compiling from source): n/a
- GCC/Compiler version (if compiling from source): n/a
- CUDA/cuDNN version:  9.0
- GPU model and memory: Quadro GP100

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with: 1. TF 1.0: `python -c "import
tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` 2. TF 2.0: `python -c
"import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**

I have a simple sequential `tf.keras` model with convolution that runs on 
a tfrecords Dataset. Suddenly, convolutions stopped working. 
No known changes made to system.

**Describe the expected behavior**
It runs as per usual

**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.

Won't help reproduce what may be system's issue.
```
ds = tf.data.TFRecordsDataset(...).shuffle(...)
model = tf.keras.Sequential([
    tf.keras.layers.Conv2D(...),
     ...
])

model.compile(...)
model.fit(...)

```

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

 name: Quadro GP100, pci bus id: 0000:af:00.0, compute capability: 6.0)
2019-09-14 08:13:43.313823: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x56244eb0db50 executing computations on platform CUDA. Devices:
2019-09-14 08:13:43.313866: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Quadro GP100, Compute Capability 6.0

2019-09-14 08:14:35.273134: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:111] Filling up shuffle buffer (this may take a while): 4255 of 129446
....

2019-09-14 08:14:36.878088: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:162] Shuffle buffer filled.
2019-09-14 08:14:48.210416: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
2019-09-14 08:14:49.011180: E tensorflow/stream_executor/cuda/cuda_dnn.cc:329] Could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR
2019-09-14 08:14:49.016488: E tensorflow/stream_executor/cuda/cuda_dnn.cc:329] Could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR
2019-09-14 08:14:49.016616: W tensorflow/core/common_runtime/base_collective_executor.cc:216] BaseCollectiveExecutor::StartAbort Unknown: Failed to get convolution algorithm. This is probably because cuDNN failed to initialize, so try looking to see if a warning log message was printed above.
	 [[{{node conv1d/conv1d}}]]
```

no warning message was printed above.

**Comment by oanush** [comment]
@SumNeuron ,
Hi,looks like the issue is with CUDA, please refer similar issue #24828 and [solution](https://github.com/tensorflow/tensorflow/issues/24828#issuecomment-457425190).Thanks!

**Comment by oanush** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
